source/views/guides/getting-started.html.slim

```diff
@@ -50,10 +50,10 @@ main
             pre.demo__example-snippet
               code
                 | [dependencies]
-                  diesel = "0.6.1"
-                  diesel_codegen = { version = "0.6.1", default-features = false, features = ["nightly", "postgres"] }
-                  dotenv = "0.8.0"
-                  dotenv_macros = "0.8.0"
+                  diesel = "0.6"
```

Comment:
I think it's probably better to leave this as is. While there won't be any major breaking changes in minor versions post 1.0, there may still be "minor breaking changes" that could cause code to fail to compile, and are allowed in minor versions as per the [API evolution guidelines](https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md).